### PR TITLE
Sort: Convert to text template

### PR DESCRIPTION
**Pull request by mintsoft**
###### Description of changes

Ref: https://github.com/duckduckgo/zeroclickinfo-goodies/issues/2767

###### People to notify (@mention interested parties)

---

Instant Answer Page: https://duck.co/ia/view/sort

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @koosha--

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@koosha-- I've gone for the text template for this for copy-pastability; an alternative option would be the list template?

**Comment by koosha--** [comment]
Thanks for the work. Unfortunately, I've been away from DDG since a long time ago and have not familiarized myself with the new codebase yet. If it works fine, I would have no objection to it.

Thanks again.

File: t/Sort.t [hunk 1]
```diff
@@ -9,72 +9,29 @@ use DDG::Test::Goodie;
...
-            input     => ['10, -1, 5.3, -95, 1, 100'],
-            operation => 'Sort descendingly',
-            result    => '100, 10, 5.3, 1, -1, -95'
-        }
-    ),
+    'sort -1, +4, -3, 5.7'   => build_test('-1, 4, -3, 5.7', 'ascendingly', '-3, -1, 4, 5.7'),
+    'sort [-1, +4, -3, 5.7]' => build_test('-1, 4, -3, 5.7', 'ascendingly', '-3, -1, 4, 5.7'),
+    'sort (-1, +4, -3, 5.7)' => build_test('-1, 4, -3, 5.7', 'ascendingly', '-3, -1, 4, 5.7'),
+    'sort desc -4.4 .5 1 66 15 -55' => build_test('-4.4, 0.5, 1, 66, 15, -55', 'descendingly', '66, 15, 1, 0.5, -4.4, -55'),
+    'sort desc -4.4 .5 1 66 2e-3 15 -55' => build_test('-4.4, 0.5, 1, 66, 0.002, 15, -55', 'descendingly', '66, 15, 1, 0.5, 0.002, -4.4, -55'),
```
Comment (GuiltyDolphin) [inline t/Sort.t]: ~~Hmm, I think we can probably get away with doing `0.5` (and similar) here; re #3123~~

This doesn't even use the Role - I'll make a separate issue for this.

**Comment by GuiltyDolphin** [comment]
@koosha-- @mintsoft Thanks! :dolphin:

**Pull request merged**